blockchain/pallets/minting/src/tests.rs

```diff
@@ -338,5 +340,39 @@ mod register_identity {
                 );
             });
         }
+
+        #[test]
+        fn register_for_minting_is_free_for_first_call() {
```

Comment:
I'd suggest looking at some of the other tests. We have functions already for registering for minting with merkle trees and other utilities for the relevant steps.

Reply:
are you referring to the `gen_tree` I have added.?I could not find a utility function which generate a new `MerkleTree` that extends the earlier one.